proto-lens-protoc/src/Data/ProtoLens/Setup.hs

```diff
@@ -233,7 +271,9 @@ copyProtosToDataDir verb root destDir files = do
         let destFile = destDir </> f
         createDirectoryIfMissingVerbose verb True
             (takeDirectory destFile)
-        installOrdinaryFile verb srcFile destFile
+        shouldCopy <- srcFile `moreRecentFile` destFile
```

Comment:
I don't think this check matters; can we remove it?  As far as I know, the installed data files don't trigger rebuilds the same way that the autogen source files are.